lib/resources/r4/financial_transaction_create.yaml

```diff
@@ -0,0 +1,300 @@
+                      "userSelected": true
+                  }
+              ],
+              "text": "United States Dollar"
+          }
+      }
+    }
+  note: Should be greater than or equal to the value of the amount. Required on payment transactions when method is Cash.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-tendered-amount
+
+- name: amount
+  required: 'Yes'
+  cardinality: 0..1
+  type: Money
+  description: The total amount of the FinancialTransaction resource.
+  action: create
+  example: |
+    {
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/amount",
+        "valueMoney": {
+            "value": "123000.00",
+            "currency": {
+                "coding": [
+                    {
+                        "system": "urn:iso:std:iso:4217",
+                        "code": "USD",
+                        "display": "United States Dollar",
+                        "userSelected": true
+                    }
+                ],
+                "text": "United States Dollar"
+            }
+        }
+    }
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/amount
+
+- name: financial-transaction-allocation
+  required: 'Yes'
+  cardinality: 1..1
+  type: Extension
+  description: How the FinancialTransaction should be applied.
+  action: create
+  example: |
+    {
+      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation",
+      "extension": [
+          {
+              "url": "target",
+              "valueReference": {
+                  "reference": "Encounter/31363178"
+              }
+          },
+          {
+              "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/amount",
+              "valueMoney": {
+                  "value": "400000.00",
+                  "currency": {
+                      "coding": [
+                          {
+                              "system": "urn:iso:std:iso:4217",
+                              "code": "USD",
+                              "display": "United States Dollar",
+                              "userSelected": true
+                          }
+                      ],
+                      "text": "United States Dollar"
+                  }
+              }
+          }
+      ]
+    }
+
+  note: Only one allocation element will be used when creating the resource for this initial release.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-allocation
+
+- name: target
+  required: 'Yes'
+  cardinality: 1..1
+  type: Reference
+  description: The specific target the FinancialTransaction should be posted to.
+  action: create
+  example: |
+    {
+      "url": "target",
+      "valueReference": {
+            "reference": "Encounter/31363178"
+      		}
+    }
+  note: Encounter will be the only posting level target that will be supported for this initial release.
```

Comment:
we should add the note about the Encounter here too.

Reply:
Are you talking about adding this note from the implementation notes

A FinancialTransaction can only be targeted at a clinical Encounter that has related financial activity within the Cerner Revenue Cycle Patient Accounting system.

or 

The financial-transaction-allocation.target field must reference an Encounter associated with an Account.

Reply:
Also should I remove the initial release part of this too?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/cf0fb07fe0fcf7b418078ee32007b3eec34065d3#diff-26870e1b94fc9955e130f3f940dbe2e7R273

Reply:
+1